src/main/java/core/basesyntax/model/FruitTransaction.java

```diff
@@ -0,0 +1,37 @@
+package core.basesyntax.model;
+
+public class FruitTransaction {

```

Comment:
Don't begin class or method implementation with an empty line. Please remove the redundant empty line at the start of the class definition.